controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/application/ApplicationApiTest.java

```diff
@@ -771,6 +786,7 @@ public void multiple_endpoints() {
         tester.computeVersionStatus();
         createAthenzDomainWithAdmin(ATHENZ_TENANT_DOMAIN, USER_ID);
         ApplicationPackage applicationPackage = new ApplicationPackageBuilder()
+                .instances("instance1")
```

Comment:
Should have one with more instances, but I guess we need instance orchestration first :)

Reply:
There's both a default and instance1 in there somewhere, but yes we need more tests when we do the orchestration part at least (but perhaps not in the API).